src/index.css

```
@@ -153,3 +153,766 @@
+}
+
+/* Bash 提示符 - 每行前面添加 $ 符号 */
+.language-bash::before,
+.language-sh::before,
+.language-shell::before,
+.language-zsh::before {
+  content: '$';
+  color: #16a34a;
+  margin-right: 0.6em;
+  font-weight: 700;
+  font-family: 'Cascadia Code', 'Fira Code', 'JetBrains Mono', 'Consolas', monospace;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)